.gitignore

```diff
@@ -35,3 +35,4 @@ load-dotenv.sh
 
 .idea/
 
+get_diff.py
```

Comment:
Why gitignore this? We probably want everyone on the team being able to use this right?